e2e/provision/playbooks/cluster.yml

```
@@ -12,6 +12,7 @@
   hosts: all
   vars:
     container_engine: docker
+    tag_real_k8s: real_k8s
```

What about to using another name?
```suggestion
    k8s_type: production
```

---

Good suggestion, I like `tag_k8s_type`. I prefer value to be `real_k8s` as against `production` for the time being.

---

After thinking about it some more, decided to keep the original name as this is more like a constant at this time and will only have the same value.

---

@liamfallon @radoslawc @arora-sagar any suggestion on this? My preference is to avoid the usage of `real` on these tags, but I'd like to hear other point of views.

---

@electrocucaracha what do you think about `tag_nonkind_k8s: nonkind_k8s` ?

---

@electrocucaracha I agree with you, even I don't like the term `real` and neither `kind` or `non-kind` because the same kind cluster we can make with minikube. 

so how about `k8s_distribution` ?

---

I am thinking better name would be pre_deployed_k8s or similar instead of nonkind_k8s. The tag identifies the code sections that need to be executed on an existing cluster and skip the other parts(installing docker, kind and such). Will push a PR later with some of the feedback once this merges as we are close to code freeze and regularly failing e2e tests are not helping the cause.